client-side/src/stories/genericInput.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import TextField from '@mui/material/TextField';
```

Comment:
please collect all the material imports to on.
like this:
import {TextField, InputAdornment } from '@mui/material';